CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ Generator:
 - Relax length precondition of `To_Context` (#1054)
 - Enable comprehensions with message sequence as target (#891)
 - Add precondition `Unitialized` to procedure `Initialize` (#788)
+- Add operators for Length and Index types (#1070)
```

Comment:
Could you please set `Length` and `Index` in backticks, so that the entry is consistent to the other entries?

Reply:
Fixed.